engines/config-query-sparql/config/rdf-join/actors-streaming.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
All of our operators are streaming, so the filename is a bit of a misnomer.
"monotonic" would be better.

But maybe we don't need this file anyways. I suspect this is here for usage in the link traversal repo,
but we won't be able to import it because link traversal has a custom rdf-join actors file.